src/index.js

```diff
@@ -54,4 +54,45 @@ document.querySelector('#task-list').addEventListener('change', (e) => {
     // Reload page
     location.reload();
```

Comment:
[Optional]
- [ ] One of the main concepts that this project wants to improve is DOM manipulation. Nowadays pages are highly dynamic and responsive. Refreshing page after every action is not a good practice. You should be able to update a specific part of your page. This is not an issue that prevents the approval of your project but  I highly recommend you to remove all page refreshes and try to update specific parts of DOM when something changes.

Reply:
Thank you for the thorough review and the good advice 👍🏼 I'll try to implement the changes needed to remove the reloading of the page and I'll keep it in mind for future projects.